make/build.js

```diff
@@ -391,6 +377,21 @@ module.exports = function build(settings, task) {
     });
     
     //////////////////////////////////////////////////////////////////////
+    // Run js-beautify for consistent coding style
+    //////////////////////////////////////////////////////////////////////
+
+    var beautify_args = [
+        "--replace",
+        "--config", "Administration/beautify.conf",
+        (src.ours).concat(cgl_mods.map(function(name) { return 'plugins/cindygl/src/js/' + name + '.js';})).filter(function(name) { return !/^build\//.test(name); }),
```

Comment:
I find this line very long and hard to read. Can you distribute the information accross several lines? Those module names occur in the CindyGL build tasks, too, so perhaps having them in some resonable location makes sense anyway.